### ISSUE TITLE
[MCP] Revise the usability study's research plan and conversation guide

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#26202]
## Issue Description
_We've made extensive design adjustments since the original research plan and conversation guide was written, so we need to review and update it to align with the new design_

---
## Tasks
- [x] _Review and update the research plan_
- [x] _Review and update the conversation guide_

## Acceptance Criteria
- [x] _The research plan and conversation guides have been reviewed and updated_
- [x] _The updated research documents have been sent to Kristen and Lauryl for approval_
- [x] _The research documents have been updated to incorporate Kristen and Lauryl's feedback_
- [x] _The research documents have been sent to Lauren, or Emily Mann if sent after June 25th, to kick off recruitment_
- [x] _The updated documents are linked in this ticket_

**Comment by rileyorr** [comment]
Becca and I finished updating the research plan and conversation guide yesterday and sent it to Kristen for review. Lauryl is OOO so we didn't send it to her, although she previously reviewed the first draft. 
This morning we incorporated Kristen's feedback and sent both documents to Lauren for approval and to kick off the recruiting process

Links to the documents in GH:

- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/Medical_Copays/research/jun-2021/research-plan.md)
- [Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/Medical_Copays/research/jun-2021/conversation-guide.md) 

I'm going to keep this ticket 'in progress' until we get the final OK from Lauren!
